Scripts/lint.sh

```
@@ -1,45 +1,101 @@
+if [ -z "$FORMAT_ONLY" ]; then
+	run_command $TOOL_CMD swift-format lint --configuration .swift-format --recursive --parallel $SWIFTFORMAT_LINT_OPTIONS Sources Tests
+	run_command $TOOL_CMD swiftlint lint $SWIFTLINT_OPTIONS
+	# Check for compilation errors
+	run_command swift build --build-tests
+fi
+
+# header.sh rewrites file headers in place, so it only runs locally — never in CI.
 if [ -z "$CI" ]; then
-	$MINT_RUN swiftformat .
-	$MINT_RUN swiftlint autocorrect
+	$PACKAGE_DIR/Scripts/header.sh -d $PACKAGE_DIR/Sources -c "Leo Dion" -o "BrightDigit" -p "ContributeWordPress"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)